res/css/photon-components.css

```diff
@@ -35,10 +35,43 @@
   margin-left: 8px;
 }
 
-.photon-button:hover:active {
```

Comment:
I didn't understand why we have both `:hover` and `:active` here. Is there a reason?